packages/xo-web/src/common/intl/messages.js

```diff
@@ -1002,9 +1003,10 @@ const messages = {
   install: 'Install',
   installPatchesTitle: 'Install patch{nPatches, plural, one {} other {es}}',
   installPatchesContent:
-    'Are you sure you want to install {nPatches, number} patch{nPatches, plural, one {} other {es}}?',
+    'This will automatically restart toolstack on each host and running VMs will not be affected. Are you sure you want to continue and install {nPatches, number} patch{nPatches, plural, one {} other {es}}?',
```

Comment:
```suggestion
    'This will automatically restart the toolstack on every host. Running VMs will not be affected. Are you sure you want to continue and install {nPatches, number} patch{nPatches, plural, one {} other {es}}?',
```